src/adapters/MeetingsJSONAdapter.js

```diff
@@ -395,6 +398,7 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     let stream;
     let deviceId;
     let perm;
+    let err;
```

Comment:
let's call this `error` (as it is returned) and rename the error in the catch block to `err`